djconnectwise/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-VERSION = (0, 3, 196, 'final')
+VERSION = (0, 3, 197, 'final')
```

Comment:
When opening PR for django-connectwise, we pretty much always bump up the version number. Keep in mind you'll need to update the corresponding number in django-topleft requirements.